Formula/b/bluez.rb

```diff
@@ -0,0 +1,33 @@
+class Bluez < Formula
+  desc "Bluetooth protocol stack for Linux"
+  homepage "https://github.com/bluez/bluez"
+  url "https://github.com/bluez/bluez/archive/refs/tags/5.72.tar.gz"
+  sha256 "0efb27303646b482518184c19ced08457f130b4ab69e3c952b9a448f986922ff"
```

Comment:
Maybe use the kernel.org tarball: https://mirrors.edge.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz 
That should have `configure` script already generated.